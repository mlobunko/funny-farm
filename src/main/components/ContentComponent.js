import React from 'react';
import Header from 'app/header/components';
import KindOfAnimal from 'app/kindOfAnimal/components';
import Cat from 'app/animals/cat/components';
import Cow from 'app/animals/cow/components';
import Dog from 'app/animals/dog/components';
import Duck from 'app/animals/duck/components';
import Rooster from 'app/animals/rooster/components';
import Sheep from 'app/animals/sheep/components';

import { StyledAnimals, StyledRow, StyledContentComponent } from './styles';

export const ContentComponent = () => (
  <StyledContentComponent>
    <Header />
    <KindOfAnimal />
    <StyledAnimals>
      <StyledRow top>
        <Duck />
        <Dog />
        <Rooster />
      </StyledRow>
      <StyledRow>
        <Cow />
        <Sheep />
        <Cat />
      </StyledRow>
    </StyledAnimals>
  </StyledContentComponent>
);

export default ContentComponent;
