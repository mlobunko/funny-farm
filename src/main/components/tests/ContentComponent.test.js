import React from 'react';
import { shallow } from 'enzyme';

import { ContentComponent } from '../ContentComponent';

describe('render', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<ContentComponent />);
    expect(wrapper).toMatchSnapshot();
  });
});
