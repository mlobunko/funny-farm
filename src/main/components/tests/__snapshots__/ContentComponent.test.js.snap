// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render renders correctly 1`] = `
<StyledContentComponent>
  <Connect(HeaderComponent) />
  <Connect(KindOfAnimalComponent) />
  <StyledAnimals>
    <StyledRow
      top={true}
    >
      <Connect(DuckComponent) />
      <Connect(DogComponent) />
      <Connect(RoosterComponent) />
    </StyledRow>
    <StyledRow>
      <Connect(CowComponent) />
      <Connect(SheepComponent) />
      <Connect(CatComponent) />
    </StyledRow>
  </StyledAnimals>
</StyledContentComponent>
`;
